quinn-proto/src/connection.rs

```diff
@@ -234,7 +228,7 @@ impl Connection {
             largest_sent_before_rto: 0,
             time_of_last_sent_retransmittable_packet: 0,
             time_of_last_sent_handshake_packet: 0,
-            largest_sent_packet: 0,
+            largest_sent_packet: !0, // Will wrap around to 0 on first transmit
```

Comment:
I find this a bit ugly. Could we do `next_packet_number: 0` instead, or do we need the largest sent packet more often than the next packet number?

Reply:
Not counting logging, `largest_sent_packet` is referenced 3 times in the recovery logic, and helps us stay close to the draft's pseudocode. I agree this is a bit ugly, though; maybe emulate the pseudocode variable with a trivial method?

Reply:
Yeah, trivial method sounds like a nice solution.